src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -9125,6 +9125,506 @@ .maxstack  2
 }");
         }
 
+        [Fact]
+        public void SkipLocalsInitAttributeOnIteratorPropagatesToItsSynthesizedMethods()
```

Comment:
>SkipLocalsInitAttributeOnIteratorPropagatesToItsSynthesizedMethods [](start = 20, length = 66)

I didn't see tests that verify that the lack of the attribute results in no behavior change, i.e. locals are initialized. Is this all covered by existing tests? Was that explicitly confirmed?

Reply:
`MoveNext` was already being covered by existing tests, but that's not the case for `GetEnumerator`. I'll add both to the tests.